client/src/pages/index.js

```diff
@@ -1,8 +1,9 @@
-import React from "react";
+import React, { Component } from "react";
```

Comment:
We don't need these changes in the master version. You can keep jacson.js as a reference, but if you could revert your changes in pages/index.js that would be great.